tests/common_setup.py

```diff
@@ -235,6 +248,117 @@ def enterprise_no_client(request):
     return env
 
 
+@pytest.fixture(scope="function")
+def enterprise_one_client(request):
+    env = container_factory.getEnterpriseSetup(num_clients=0)
+    request.addfinalizer(env.teardown)
+
+    env.setup()
+    reset_mender_api(env)
+
+    tenant = create_tenant(env)
+    mender_device = new_tenant_client(env, "mender-client", tenant["tenant_token"])
+    mender_device.ssh_is_opened()
+    env.device = mender_device
```

Comment:
`new_tenant_client` already adds the device to the environment, so these lines (and similarly below) could be:
```suggestion
    new_tenant_client(env, "mender-client", tenant["tenant_token"])
    env.device_group.ssh_is_opened()
```

Reply:
I will address it in the other PR (https://github.com/mendersoftware/integration/pull/1923/commits/f20c0e80846fb2a1600903e2f28b726b909dcbf3)